app/static/openapi.yaml

```diff
@@ -962,11 +965,11 @@ components:
             {
               'apiVersion': '1.0',
               'data': null,
-              'errors': [
-                  {
-                      'code': 'unauthorized'
-                  }
-              ],
+              'errors': {
+                'invalid_credentials': {
+                  'message': 'The email or password you submitted is incorrect'
```

Comment:
The 401 response sometimes looks like this:
```json
{
    "apiVersion": "1.0",
    "data": null,
    "errors": {
        "unauthorized": {
            "message": "You must provide a valid API token in the x-apikey header."
        }
    },
    "status": "Unauthorized",
    "status_code": 401
}
```
I don't think that we can reuse this same 401 response example for all of the endpoints anymore. We need to have this `invalid_credentials` example for the Request Key endpoint and the one I pasted above for the Create/Update Resource endpoints